tests/contrib/validator/test_validate_ret_model.py

```diff
@@ -0,0 +1,20 @@
+from hypothesis import given
+from hypothesis.strategies import sampled_from
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.linear_model import LogisticRegression
+from sklearn.pipeline import Pipeline
+
+from deployme.contrib.supported import ModelType
+from deployme.contrib.validator import validate_ret_model
+
+
+@given(model=sampled_from([RandomForestClassifier(), LogisticRegression()]))
+def test_validate_ret_model_valid(model):
+    """Ensures that valid model is returned."""
+    assert validate_ret_model(model) == ModelType.SKLEARN_MODEL
+
+
+@given(model=sampled_from([Pipeline(...)]))
+def test_validate_ret_model_valid_pipeline(model):
+    """Ensures that valid pipeline is returned."""
+    assert validate_ret_model(model) == ModelType.SKLEARN_PIPE
```

Comment:
What about testing with LAMA model given?

Reply:
Now we don't have official LAMA support, need to make new descriptive issue for LAMA support. But I want to emphasize that LightAutoML should not be in the official dependencies, because, probably, not every person will need it (By the way, this should also apply to the docker dependency, in my opinion). I also want to have optional tests for this case.